### DISCUSSION TITLE
【羊村】木头羊语录｜ApacheCN

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/home#215] [Ideas]
本个issues用于收录木头羊语录，请大家积极参与收集！

**Comment by MilkSU-Official** [comment]
![image](https://user-images.githubusercontent.com/17907135/118214231-f2224a80-b4a1-11eb-9826-2b93797c3e86.png)

**Comment by shang9557** [comment]
![image](https://user-images.githubusercontent.com/34467188/118215128-8ccf5900-b4a3-11eb-86ad-6f75b8f9a9a7.png)

**Comment by jiangzhonglian** [comment]
![image](https://user-images.githubusercontent.com/9199175/118215802-dec4ae80-b4a4-11eb-8f9f-bf9019bd8083.png)

**Comment by fellowing** [comment]
![514](https://user-images.githubusercontent.com/43286388/118215983-36fbb080-b4a5-11eb-94a5-af00322e0e4f.jpg)

**Comment by PandaK404** [comment]
![QQ图片20210514112741](https://user-images.githubusercontent.com/83934795/118217118-69a6a880-b4a7-11eb-8374-be944733625e.png)

**Comment by jiangzhonglian** [comment]
![image](https://user-images.githubusercontent.com/9199175/118232840-ba2bff00-b4c3-11eb-801b-7561655ba4a8.png)

**Comment by jiangzhonglian** [comment]
![image](https://user-images.githubusercontent.com/9199175/118232880-c748ee00-b4c3-11eb-8c31-34e80143b69e.png)

**Comment by yz-xlame** [comment]
![QQ图片20210517111524](https://user-images.githubusercontent.com/16894963/118428122-51ca6100-b701-11eb-8705-a0135f727cdd.png)
![QQ图片20210517111530](https://user-images.githubusercontent.com/16894963/118428126-568f1500-b701-11eb-95c1-cf55694b54b1.png)
![%HG5HDOSJEL$ZIR{ JG_AZ6](https://user-images.githubusercontent.com/16894963/118457766-aafbba00-b72c-11eb-9124-8cf78c4c6eda.png)

**Comment by yz-xlame** [comment]
![956WOUIR86YYT6ILA2@%)XG](https://user-images.githubusercontent.com/16894963/118429029-5728ab00-b703-11eb-9569-7b12c9831b27.png)

**Comment by linnnff** [comment]
![image](https://user-images.githubusercontent.com/41185967/118603365-659bc300-b7e6-11eb-9071-3ccbbd8744d8.png)

**Comment by yz-xlame** [comment]
![NRKM2 IVRA(B0WGIMR8)4S5](https://user-images.githubusercontent.com/16894963/118933657-0bcd0180-b97c-11eb-8416-16b6ccf17592.png)

**Comment by linnnff** [comment]
![image](https://user-images.githubusercontent.com/41185967/118955308-28bfff80-b991-11eb-8e2b-5213c248d07c.png)

**Comment by yz-xlame** [comment]
![KLD {R{IZXNV(W Y2CX@5_V](https://user-images.githubusercontent.com/16894963/119074007-d3cbca00-ba20-11eb-96d2-66e14b3c3376.png)

**Comment by kviccn** [comment]
![image](https://user-images.githubusercontent.com/17717641/120170681-cb9b3800-c233-11eb-8686-97bbf9ca2bc6.png)

**Comment by kviccn** [comment]
![image](https://user-images.githubusercontent.com/17717641/120170711-d48c0980-c233-11eb-9e58-d80a85664498.png)

**Comment by kviccn** [comment]
![image](https://user-images.githubusercontent.com/17717641/120170744-de157180-c233-11eb-8d19-41f05d7e59e3.png)

**Comment by kviccn** [comment]
![image](https://user-images.githubusercontent.com/17717641/120310445-fc00d600-c308-11eb-8c9a-caa3227b2b44.png)
![image](https://user-images.githubusercontent.com/17717641/120310469-00c58a00-c309-11eb-99f4-0ef032e12a04.png)

**Comment by yanqiang921106** [comment]
![image](https://user-images.githubusercontent.com/34933285/120445094-db915400-c3ba-11eb-878e-55492f6f49c2.png)

**Comment by yanqiang921106** [comment]
![image](https://user-images.githubusercontent.com/34933285/120755258-520b8e80-c540-11eb-8ff9-d421e9503d83.png)

**Comment by yanqiang921106** [comment]
![image](https://user-images.githubusercontent.com/34933285/120755607-c47c6e80-c540-11eb-9235-610f43dc5e6a.png)

**Comment by yz-xlame** [comment]
![0{ X25ERUG0 Z@CBO1U6A{W](https://user-images.githubusercontent.com/16894963/122963678-159db680-d3b9-11eb-83fd-94f8984497b9.png)

**Comment by yz-xlame** [comment]
![$YA%EURME7@NAGAZAP3R(12](https://user-images.githubusercontent.com/16894963/122963854-3f56dd80-d3b9-11eb-9a62-1412f27113c3.png)

**Comment by yz-xlame** [comment]
![image](https://user-images.githubusercontent.com/16894963/122964035-7200d600-d3b9-11eb-8b50-5fc1d31b7099.png)

**Comment by yz-xlame** [comment]
![image](https://user-images.githubusercontent.com/16894963/122964465-e176c580-d3b9-11eb-9401-2b47494685ae.png)

**Comment by Mintonmu** [comment]
![SmartSelect_20210623-103302_QQ](https://user-images.githubusercontent.com/28354069/123026321-8622f280-d40e-11eb-985e-bb3840ad2f8e.jpg)

**Comment by yz-xlame** [comment]
![(7T$1D53ZVTD2R()5)V2R1W](https://user-images.githubusercontent.com/16894963/123513446-e2606d80-d6bf-11eb-963e-92df4c4002bf.png)
![CG %{J7(3U()O( OIJ![(7T$1D53ZVTD2R()5)V2R1W](https://user-images.githubusercontent.com/16894963/123513446-e2606d80-d6bf-11eb-963e-92df4c4002bf.png)
03D](https://user-images.githubusercontent.com/16894963/123513450-e68c8b00-d6bf-11eb-95e3-39464c4bd77e.png)

**Comment by yz-xlame** [comment]
![@JLM _J%80 @BWE60ELZ{}D](https://user-images.githubusercontent.com/16894963/123534519-3d41a580-d750-11eb-9148-09565212fe1e.jpg)

**Comment by yz-xlame** [comment]
![3 2Y@4A)@ROHRHW BEOX_O](https://user-images.githubusercontent.com/16894963/123540101-de8c2400-d76f-11eb-9317-8f4c2a29f686.png)

**Comment by yz-xlame** [comment]
![@8Q%1XGAY%BBW3ETCAIG`GG](https://user-images.githubusercontent.com/16894963/123540120-f1065d80-d76f-11eb-8800-4213bc7ed262.png)

**Comment by kviccn** [comment]
![image](https://user-images.githubusercontent.com/17717641/123775934-63f30e00-d901-11eb-8444-4cbd5fb87fad.png)

**Comment by zzkOo** [comment]
![Screenshot_20210629_175344_com tencent mobileqq](https://user-images.githubusercontent.com/83104951/123779044-568b5300-d904-11eb-8514-4e08336cb349.png)

**Comment by limouren007** [comment]
![@ }1G7TP)XT}AJ 3~F(MHCM](https://user-images.githubusercontent.com/72721929/123897047-5c7d4480-d995-11eb-8d64-1c7a8355c68d.png)

**Comment by limouren007** [comment]
![53FK 4Z I{{U8B%57 R8)R5](https://user-images.githubusercontent.com/72721929/123957767-c110c180-d9de-11eb-9e85-01671c1ddd32.png)
![UFE4NZ(I8PQA@OQ29EPEX~F](https://user-images.githubusercontent.com/72721929/123957775-c3731b80-d9de-11eb-82a6-401d0e9087af.png)
![6DJ}SMK8X(VESF3@9(UH_Y2](https://user-images.githubusercontent.com/72721929/123957785-c79f3900-d9de-11eb-8628-1bdad6aece49.png)

**Comment by limouren007** [comment]
![YOQOT0SY(CVD4`9@%EQORMA](https://user-images.githubusercontent.com/72721929/123957984-16e56980-d9df-11eb-89f2-3746345f3ae5.png)
![GWKVTJ{9)V}} 33CAP6 1`L](https://user-images.githubusercontent.com/72721929/123957996-1947c380-d9df-11eb-9b0e-03a5caba9e21.png)

**Comment by limouren007** [comment]
![~G0`E%8Y5IG}$18QJW_YZEW](https://user-images.githubusercontent.com/72721929/123958276-704d9880-d9df-11eb-8a85-aedcdfe8cbd7.jpg)

**Comment by limouren007** [comment]
![YU6}JRX%UHG0Z1(66B_VCIB](https://user-images.githubusercontent.com/72721929/123958426-a3902780-d9df-11eb-9c7e-08bb2908edd9.png)

**Comment by yz-xlame** [comment]
![TJ4@BG4EI$B) (3D%$)DU34](https://user-images.githubusercontent.com/16894963/123987257-9d0ea980-d9f9-11eb-8a14-3d2ac1dfd00b.png)

**Comment by yz-xlame** [comment]
![image](https://user-images.githubusercontent.com/16894963/123987439-c3344980-d9f9-11eb-93c9-f34af77cb384.png)

![image](https://user-images.githubusercontent.com/16894963/123987373-b57ec400-d9f9-11eb-973f-3422bf627127.png)

**Comment by yz-xlame** [comment]
![image](https://user-images.githubusercontent.com/16894963/123987700-fe367d00-d9f9-11eb-8172-07966210c363.png)

**Comment by limouren007** [comment]
![FNM @FRX(W_B1OH0WW)QGVV](https://user-images.githubusercontent.com/72721929/124092387-5ff1f800-da89-11eb-87b9-7bf7853dab1c.png)

**Comment by 1960308383** [comment]
![Cache_7ed26cba7878d06e](https://user-images.githubusercontent.com/58506294/124708236-94076600-df2c-11eb-9f63-bc5a08080f50.jpg)

**Comment by 1960308383** [comment]
![Cache_414eeb2a5dd909df](https://user-images.githubusercontent.com/58506294/124708249-99fd4700-df2c-11eb-8211-b23709f3c46a.jpg)

**Comment by 1960308383** [comment]
![Cache_3f66c56517e4844b](https://user-images.githubusercontent.com/58506294/124708601-12640800-df2d-11eb-829c-cd3064fb600b.jpg)

**Comment by 1960308383** [comment]
![Cache_715eeed81a541aa6](https://user-images.githubusercontent.com/58506294/124708631-1b54d980-df2d-11eb-84c9-d841cc85f289.jpg)

**Comment by 1960308383** [comment]
![Cache_722899f6e02bbf3b](https://user-images.githubusercontent.com/58506294/124708663-2871c880-df2d-11eb-8a1e-ba74cc76ff13.jpg)

**Comment by yz-xlame** [comment]
![P8}7LQKFI1(~7NMRR7 )4NT](https://user-images.githubusercontent.com/16894963/126898785-af25124b-da84-4b44-9438-c318de1da51c.png)
![S{~H02CEPX JZ8}5QARG0Y](https://user-images.githubusercontent.com/16894963/126898789-e5aa08fc-c107-4e98-bc86-b5f2399fe5c7.png)
![44D}UVDMJ}AFC5SDZOM17MI](https://user-images.githubusercontent.com/16894963/126898791-0a9eeb34-c866-4472-8782-e90edb402713.png)
![85( SQ27Y992}8R%H8H8H W](https://user-images.githubusercontent.com/16894963/126898792-afc29ec8-896f-4a54-ba3b-731bbebd7851.png)
![~D_Z1L)7DT0_X 3$76O${}8](https://user-images.githubusercontent.com/16894963/126898793-492cfde9-0540-4ac9-875f-d23aca68a877.jpg)
![T_DPGV2ON`WT(09TS02)EFJ](https://user-images.githubusercontent.com/16894963/127013785-e53ec70f-d70c-4921-8931-b9ef88a5e35f.png)

**Comment by QTNiCheng** [comment]
<img width="323" alt="}8)B9KNCM2CXRP I{)`(GV7" src="https://user-images.githubusercontent.com/19570475/128300631-4b3c277e-d876-4d0e-9792-04a2a041a75e.png">

**Comment by fellowing** [comment]
![UFYBY32 RJE7S6S3}( TN)N](https://user-images.githubusercontent.com/43286388/129726469-36876681-138b-4ea0-ae29-93f18d6d1354.jpg)

**Comment by 1960308383** [comment]
![null-21c19fcb83fbce1c](https://user-images.githubusercontent.com/58506294/129840030-11f10383-cdb1-4dfe-862a-69de714b7546.jpg)

**Comment by 1960308383** [comment]
![E1718EAB9FDB4790810638F61DEE959D](https://user-images.githubusercontent.com/58506294/130322347-12ed16fc-eae6-4806-8e34-90dfb512d211.png)

**Comment by 1960308383** [comment]
![image](https://user-images.githubusercontent.com/58506294/130322356-3a57e4bb-0773-4714-9ac8-5b6b73eca7dc.png)

**Comment by 1960308383** [comment]
![image](https://user-images.githubusercontent.com/58506294/130322367-89bed08d-f7a1-4130-951b-e8877f6c6a58.png)

**Comment by fellowing** [comment]
![20210826215805](https://user-images.githubusercontent.com/43286388/130987542-ac90f5e4-b19c-48ca-8891-0f1417d0780f.jpg)

**Comment by fellowing** [comment]
![微信图片_20210829212730](https://user-images.githubusercontent.com/43286388/131252099-75daae1c-5e16-4b5b-ae86-7929a3e44381.jpg)

**Comment by fellowing** [comment]
![微信图片_20210830202211](https://user-images.githubusercontent.com/43286388/131338161-977b8e2c-63ca-42f7-912e-b37a5dc5f3dd.jpg)

**Comment by 1960308383** [comment]
![Cache_5d67fd1a7e712f3c](https://user-images.githubusercontent.com/58506294/131338822-dc7479d5-4ac4-49b8-90bc-8e7a8a25c86d.jpg)
![Cache_-581a1364e1e51a95](https://user-images.githubusercontent.com/58506294/131338836-b04038a4-fcff-42f7-ba5f-669e5337f653.jpg)

**Comment by 1960308383** [comment]
![Cache_6aad4b5d01a18b81](https://user-images.githubusercontent.com/58506294/131338863-d11548db-9cb7-4542-ac26-13a2ccb68b69.jpg)

**Comment by 1960308383** [comment]
![null-7c1e2bf1a0a6f9c4](https://user-images.githubusercontent.com/58506294/131456151-223e4ee7-db3a-4ef9-a8be-278856f8adbb.jpg)
![null1d2697893a5d2d87](https://user-images.githubusercontent.com/58506294/131456161-0c2d30b3-2501-49e5-ac6c-acba89cf3bd1.jpg)

**Comment by wizardforcel** [comment]
![IMG_20210904_152945](https://user-images.githubusercontent.com/5080126/132086822-e8ccdbfb-7845-40da-a8f0-085fc56097cc.jpg)

**Comment by fellowing** [comment]
![微信图片_20210921170526](https://user-images.githubusercontent.com/43286388/134144084-6a720abf-4e4c-49c7-97af-9554b2a81fdc.png)

**Comment by 1960308383** [comment]
![EA49FCFFA1136547AA72B8E7E6391D79](https://user-images.githubusercontent.com/58506294/134297734-da45b440-240b-4c6a-8566-2c747519304e.jpg
![5E1BAD8B86B4977A80B8E25005A0B6FA](https://user-images.githubusercontent.com/58506294/134297738-b2d09f47-4f06-4607-b858-de58e1ec38e3.png)
![5A926AB56D5D9950ED28B973CF1DDE48](https://user-images.githubusercontent.com/58506294/134297746-20fa5967-d6a1-44ea-8ff0-ba628398a913.jpg)
)
![CBD6BE8264C93EB33C17081F0BA4FA49](https://user-images.githubusercontent.com/58506294/134297754-04c627d0-4449-4beb-ac43-aaa109650279.png)

**Comment by 1960308383** [comment]
![B846BC085315BBECE3C181CCF14398FB](https://user-images.githubusercontent.com/58506294/134297823-7be81b6d-50ad-4c1b-952b-a3bc93592385.jpg)
一开始说打四个，结果第一个就。。。
![951A920DD23D9376D926E7356DB0B8C7](https://user-images.githubusercontent.com/58506294/134297907-48189349-6b3f-4ea5-8169-6ed006f8482b.png)
。

**Comment by fellowing** [comment]
![a6928caf2e08a1acab3ef3672b9e9f4](https://user-images.githubusercontent.com/43286388/135067298-63ec0b14-b019-4283-94e1-7cbf3cde6b83.png)

**Comment by fellowing** [comment]
![d63438f6fd52ab4c276067c04ef7eb6](https://user-images.githubusercontent.com/43286388/135067318-9897c749-be82-4629-a629-b95bb720213d.png)

**Comment by oneheartding** [comment]
![QQ图片20211022112034](https://user-images.githubusercontent.com/22575730/138388187-2c8abef2-a099-45c0-a0e9-2dbeb9ea0c2b.png)

**Comment by zzkOo** [comment]
![Screenshot_20211025_214449_com tencent mobileqq](https://user-images.githubusercontent.com/83104951/138707887-10929265-c119-44e1-9211-64b6d08e0fef.png)

**Comment by leader402** [comment]
和望之聊天系列

![image](https://user-images.githubusercontent.com/47787078/146200207-3d4e3a31-3eb9-4f2d-ac18-87884710695e.png)

**Comment by leader402** [comment]
和望之聊天系列2
![6{BOD)J9 EDLE7ST DVXICE](https://user-images.githubusercontent.com/47787078/146200377-807b5ad8-0c6c-43ea-9cbe-070e4d43dd85.png)

**Comment by leader402** [comment]
和望之聊天系列3
![image](https://user-images.githubusercontent.com/47787078/146200709-a7ff4677-9cdf-49a1-b1cc-e1e6333e04bc.png)

**Comment by leader402** [comment]
和望之聊天系列4
![image](https://user-images.githubusercontent.com/47787078/146201254-8a830148-a6cb-4c9e-9d84-ca18d5cb1ebf.png)

**Comment by MeteorsHeart** [comment]
![image](https://user-images.githubusercontent.com/76858595/146640313-85bfdb3e-ea0d-4a66-8b44-a7dc73853a71.png)

**Comment by linnnff** [comment]
![G57Z}`I~4J~~SJ60)}~QG_L](https://user-images.githubusercontent.com/41185967/147639689-89eaecb5-c38c-4553-a192-51809036f92e.png)

**Comment by jiangzhonglian** [comment]
<img width="461" alt="image" src="https://user-images.githubusercontent.com/9199175/148043181-d60185f4-2e49-4b59-9327-8efc233ead7c.png">

**Comment by jiangzhonglian** [comment]
![image](https://user-images.githubusercontent.com/9199175/148043213-5912b022-0fd8-48be-b7f5-556d2cd5336f.png)

**Comment by jiangzhonglian** [comment]
<img width="675" alt="image" src="https://user-images.githubusercontent.com/9199175/148043460-2f4d8fb2-3ebc-492d-a499-7bd937c2c3b8.png">

**Comment by oneheartding** [comment]
![-46689d1585f934a.png](https://user-images.githubusercontent.com/22575730/154796838-473bc272-4f5d-4808-9be3-936d356a9b6a.png)

**Comment by oneheartding** [comment]
![QQ图片20221212130759](https://user-images.githubusercontent.com/22575730/206965300-6596c544-5aeb-46c8-908c-aea7b690ca95.jpg)

**Comment by jiangzhonglian** [comment]
![image](https://user-images.githubusercontent.com/9199175/207493970-e83aca99-a34b-4e8f-abe6-49199cad60c4.png)

**Comment by linnnff** [comment]
![4U{B$R8Q9PVXH %@~9`@}(0](https://user-images.githubusercontent.com/41185967/230812108-bf632115-8134-465c-9ea2-48a70878cd63.png)

**Comment by jiangzhonglian** [comment]
1
